client/src/components/ReviewList.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Review from './Review.jsx';
+import styled from 'styled-components';
+
+
+
+// const Container = styled.container`
+//   display: grid;
+//   grid-template-columns: 1fr 2fr;
+// `  
+// const ReviewItem = styled.item`
+//   display: grid;
+//   grid-template-columns: 1fr 2fr;
+// `  
```

Comment:
this still in progress or old code to remove?

Reply:
i'll delete that. early experimentation